lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -740,6 +797,88 @@ ParseResult FIRParser::parseOptionalRUW(RUWAttr &result) {
   return success();
 }
 
+ParseResult FIRParser::importAnnotations(const SMLoc &loc,
+                                         StringRef annotationsStr,
+                                         MLIRContext *context,
+                                         StringRef errorMsgParse) {
```

Comment:
I don't think you need to pass in errorMsgParse since you have location info that makes it obvious what sort of thing you're parsing.  I'd forward substitute it to simplify the code a bit.

Reply:
I'll opt for a more a generic message here that covers both the file and inline annotation situation. This can be improved, e.g., with an actual SMLoc pointing at the annotation file at some later point. Changed in a `!fixup` commit.